app/components/PlatformCards/DiscordPlatform.tsx

```diff
@@ -233,9 +233,9 @@ export default function DiscordCard(): JSX.Element {
       isClosable: true,
       render: (result: any) => (
         <DoneToastContent
-          title="Verificaton Failed"
+          title="Verification Failed"
           body="Please make sure you fulfill the requirements for this stamp."
-          icon="../../assets/whiteBgShieldExclamation.svg"
+          icon="../../assets/verification-failed.svg"
```

Comment:
The error image wasn't showing for me. I think we need to add a default height to the image in the in [DoneToastContent](https://github.com/gitcoinco/passport/blob/9a386249a39f0101bd670d958d5051f325a129f8/app/components/DoneToastContent.tsx#L34)

with h-6 added to the class I was able to get it to render with ` className="sticky top-0 h-6"`